terraform/features-api/init.tf

```diff
@@ -0,0 +1,24 @@
+provider "aws" {
+  alias  = "west1"
+  region = "us-west-1"
+}
+
+provider "aws" {
+  alias  = "west2"
+  region = "us-west-2"
+}
+
+terraform {
+  required_version = "1.4.6"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.0"
+    }
+  }
+  backend "s3" {
+    bucket = "veda-wfs3-tf-state-bucket"
```

Comment:
you'll need to manually create a new S3 bucket in this region and change this name so we don't clobber this state bucket